client/web/templates/conditionsByDrug.html

```diff
@@ -1,31 +1,30 @@
 <div class="col-xs-12">
     <div class="row" style="margin-top:20px">
-        <div class="col-xs-1">
-        </div>
-        <div class="col-xs-10" style="margin-top:20px">
+        <div class="col-xs-offset-1 col-xs-10" style="margin-top:20px">
             <h2><span class="text-capitalize">{{appModel.treatment.drug_concept_name}}</span> Condition List from Product Label</h2><br>
+            <h4>Amongst patients taking <span class="text-capitalize">{{appModel.treatment.drug_concept_name}}</span>,
+                onset of the following conditions occurs during the 1 year after starting the drug</h4>
             <div ng-if="!!!evidence.length" class="alert alert-info">
                 <strong>Sorry!</strong> Product label not found for the selected drug.
             </div>
+            <br>
             <table ng-if="!!evidence.length" class="table table-striped table-hover">
                 <thead>
                 <tr>
                     <th>Condition</th>
-                    <th>Lowest Incidence Proportion</th>
-                    <th>Highest Incidence Proportion</th>
+                    <th>Lowest Incidence (%)</th>
+                    <th>Highest Incidence (%)</th>
                 </tr>
                 </thead>
                 <tbody>
                 <tr ng-repeat="r in evidence | limitTo:itemsLimit track by $index">
                     <td><a ui-sref="app.drug_condition({ drugConceptId: appModel.treatment.drug_concept_id, conditionConceptId: r.outcome_concept_id })">{{r.outcome_concept_name}}</a></td>
-                    <td>{{r.incidence_proportion_range_low | number : 4}}</td>
-                    <td>{{r.incidence_proportion_range_high | number : 4}}</td>
+                    <td align="right">{{r.incidence_proportion_range_low * 100 | number : 2}}</td>
```

Comment:
This looks kind of weird on my screen (screenshot below); if the problem is that the columns are too wide, would giving the table `class="col-md-6"` help?

![image](https://user-images.githubusercontent.com/8049605/34592970-de8bc2f8-f195-11e7-905a-3f31876e45e7.png)

Reply:
@mark-velez - I changed numbers to 2 decimal places from 4 and to right aligned from center aligned. When we showed 4 decimal places it looked fine with center aligned but after I show 2 decimal places it looks bad with center aligned so I had to change to right aligned. Then it looks weird. It might help to reduce the column width by adding  `class="col-md-6"`

Reply:
@juh7007 I just committed change to reduce table width, does it look ok to you? (screenshot below)

![image](https://user-images.githubusercontent.com/8049605/34622231-5e22f2d2-f21a-11e7-94cf-4af1ef89eb80.png)